playground-next13-rsc/app/page.tsx

```diff
@@ -2,9 +2,9 @@ import type {
   ISbStoriesParams,
   StoryblokClient,
 } from '@storyblok/react/rsc';
+import { getStoryblokApi } from '@/lib/storyblok';
 import {
-  getStoryblokApi,
-  StoryblokComponent,
+  SbServerComponent,
```

Comment:
@edodusi Is this maybe `StoryblokServerComponent`? (same on the following lines)

Reply:
totally